src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -24,6 +24,14 @@ internal static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Avx.IsSupported ? Vector256Alignment : (Sse.IsSupported ? Vector128Alignment : FloatAlignment);
 
+        /// <summary>
+        /// Multiplies a matrix times a source.
+        /// </summary>
+        /// <param name="transpose">Boolean to transpose the matrix or not.</param>
```

Comment:
💡 I would typically write this using the following language:

```suggestion
        /// <param name="transpose"><see langword="true"/> to transpose the matrix; otherwise, <see langword="false"/>.</param>
```

Reply:
TIL there's another attribute on the `see` element besides `href`...